perllib/FixMyStreet/App/Controller/Waste/Bulky.pm

```diff
@@ -108,7 +108,13 @@ sub index : PathPart('') : Chained('setup') : Args(0) {
     $c->forward('form');
 
     if ( $c->stash->{form}->current_page->name eq 'intro' ) {
-        $c->cobrand->call_hook(clear_cached_lookups_bulky_slots => $c->stash->{property}{uprn});
+        my $id_for_cache
```

Comment:
Looking at master Peterborough code, there is some other confusion as `clear_cached_lookups_property` is called with both an ID and a URPN, so some clearing of the bulky slots isn't working anyway when called from there rather than directly like here.

I think there is a solution, which is change the setting of available_bulky_slots in Peterborough's `find_available_bulky_slots` to use property->{id} rather than {uprn}. Then everything is using ID for the key, not UPRN, and everything to do with clear_cached_lookups* can only use ID and not worry about the UPRN at all, including here